packages/evolution-frontend/src/components/pages/auth/AuthPageHOC.tsx

```diff
@@ -72,6 +72,17 @@ const authPageHOC = <P extends AuthPageProps & WithTranslation>(WrappedComponent
             setCurrentAuthMethod(authMethod);
         };
 
+        // Check if there are no auth methods (except directToken)
+        const hasNoAuthMethods = () => {
```

Comment:
This can be a constant instead of function, the value will not change. You can use React.useMemo()

Reply:
@tahini 
Done. 

First time using React.useMemo(). Do we do that for performance?

Reply:
Yes, I tested every possible method. There was one issue (https://github.com/chairemobilite/evolution/issues/678), it was probably there before this commit.